pre-commit

```diff
@@ -25,9 +28,8 @@ else
     echo ""
     echo "Please install php-cs-fixer, e.g.:"
     echo ""
-    echo "  composer install"
+    echo "${YELLOW}  composer install${NO_COLOR}"
```

Comment:
Why yellow?

Reply:
why not ? :laughing:

Reply:
If the text was yellow as a warning, sure. But it makes the "do this" part look like a warning rather than an instruction.